src/basecamp.h

```diff
@@ -2,36 +2,80 @@
 #ifndef BASECAMP_H
 #define BASECAMP_H
 
+#include "enums.h"
+#include "json.h"
+#include "npc.h"
```

Comment:
This seems unnecessary (you can forward declare the `npc` class).

Same with "json.h" (forward declare the json classes).

Reply:
Fixed, thanks.